.Rbuildignore

```diff
@@ -2,3 +2,4 @@
 ^renv\.lock$
 ^.*\.Rproj$
 ^\.Rproj\.user$
+^tests/upgrades/
```

Comment:
wouldn't that match just a theoretical `upgrades` file in the `tests` folder, since `$` just matches end of the line?

I thought alternatively about doing something like

```
^tests/upgrades/*\.jasp$
```
or just
```
^tests/upgrades/*$
```
but the latter actually does basically the same as what is there right now, or am I mistaken?

Reply:
> wouldn't that match just a theoretical `upgrades` file in the tests folder, since $ just matches end of the line?

yes, that is the idea. There cannot be a filename AND directory with the name `upgrades` btw, at least not on Unix platforms. By specifying `^tests/upgrades/` it could be that you only exclude a folder with path `tests/upgrades` but not a file with path `tests/upgrades` although I'm not sure.

> but the latter actually does basically the same as what is there right now, or am I mistaken?

yes, except that it's a bit more inefficient (it will match every file in tests/upgrades, rather than just matching tests/upgrades).

Note that the default `^\.Rproj\.user$` is also a directory, yet RStudio does not do ^\.Rproj\.user/$. As long as this works I don't really care though.

Reply:
ok thanks!